app/allocation/controllers/view.test.js

```diff
@@ -0,0 +1,105 @@
+const presenters = require('../../../common/presenters')
+const filters = require('../../../config/nunjucks/filters')
+
+const handler = require('./view')
+
+describe('view allocation', function() {
+  let locals
+  let t
+  let render
+  let output
+  beforeEach(function() {
+    sinon.stub(filters, 'formatDate').returns('__date__')
+    sinon.stub(filters, 'formatTime').returns('__time__')
+    sinon.stub(filters, 'formatDateAsRelativeDay').returns('__relative_date__')
```

Comment:
The filters don't appear to be part of the view controller, do we need to import and stub these methods?